android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -157,8 +158,26 @@ public void sendToNotificationCentre(Bundle bundle) {
                 title = context.getPackageManager().getApplicationLabel(appInfo).toString();
             }
 
-            NotificationCompat.Builder notification = new NotificationCompat.Builder(context)
-                    .setContentTitle(title)
+            NotificationManager notificationManager = notificationManager();
+
+            NotificationCompat.Builder notification;
+
+            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+                notification = new NotificationCompat.Builder(context);
+            } else {
+                String channelId = "step_reminders_channel_id";
+
+                NotificationChannel channel = new NotificationChannel(channelId,
+                        "Steps",
+                        NotificationManager.IMPORTANCE_DEFAULT);
+                channel.setDescription("This channel is used for Step Reminders.");
+                notificationManager.createNotificationChannel(channel);
+
+                notification = new NotificationCompat.Builder(context, channelId)
+                    .setChannelId(channelId);
```

Comment:
Ah, thanks for pointing that out.  But I think I need to run `NotificationCompat.Builder(context, channelId)` here since the version of that constructor without the channel ID is deprecated and we can't run the version with the ID in pre-O.

Reply:
OK, thanks.  I'll make sure to fix those things before this gets merged.  I want to make sure we're OK to name the channel "Steps."  Also, I realize I'm hardcoding this, but I figure I might as well since for now we have to fork the library anyway...

Reply:
yeah, I noticed the hardcoding too ;)

the constructor without the channelId is actually deprecated as of 26.1.0 https://developer.android.com/reference/android/support/v4/app/NotificationCompat.Builder.html#NotificationCompat.Builder(android.content.Context)